components/AuthForm/VerifyEmailForm.tsx

```
@@ -0,0 +1,108 @@
+  return (
+    <div className="text-right">
+      <h2 className="mt-4 text-lg font-bold text-center mb-2">
+        Verify your email
+      </h2>
+      <p className="text-center text-medium mb-4">We sent a code to {email}</p>
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <Input
+          errorMessage={errors?.code?.message || ""}
+          isInvalid={!!errors.code}
+          label="Enter your verification code"
+          variant={!!errors.code ? "bordered" : "flat"}
```

Biome

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)